accounts/abi/bind/backends/simulated.go

```diff
@@ -675,14 +675,17 @@ func (b *SimulatedBackend) SubscribeNewHead(ctx context.Context, ch chan<- *type
 }
 
 // AdjustTime adds a time shift to the simulated clock.
+// You have to call `b.Commit()` right after.
```

Comment:
is there a case in which `Commit` should not be called? Because if so, then just call it at the end, so that people don't forget

Reply:
Yeah thats a good idea, I didn't do it, because commit would need the mutex, but I can also change it

Reply:
Ahh nvm the comment was wrong. You can add transactions on top of it, and they will have the new time. If you then call commit than they will be committed as normal